### ISSUE TITLE
Warning: Failed prop type: Invalid prop `listRef` of type `object` supplied to `PagingControls`, expected `string`.

### ISSUE THREAD

**Issue by alexmiddeleer** [mythgard-hub/mythgard-hub#228]
Seems like on some of the pages that have paging, we messed up either the inputs or proptypes. There doesn't seem to be anything special to reproduce this, it always happens. It doesn't cause a problem though, this is more of a polish fix.

```
Warning: Failed prop type: Invalid prop `listRef` of type `object` supplied to `PagingControls`, expected `string`.
Warning: Failed prop type: Invalid prop `listRef` of type `object` supplied to `PagingControls`, expected `string`.
    in PagingControls (created by CardList)
    in CardList (created by AllCards)
    in AllCards (created by SomeCards)
    in SomeCards (created by DeckBuilderCardDisplay)
    in div (created by DeckBuilderCardDisplay)
    in div (created by DeckBuilderCardDisplay)
    in DeckBuilderCardDisplay (created by DeckBuilderPage)
    in div (created by DeckBuilderPage)
    in div (created by DeckBuilderPage)
    in div (created by Layout)
    in Layout (created by withRouter(Layout))
    in withRouter(Layout) (created by DeckBuilderPage)
    in DeckBuilderPage (created by MyApp)
    in ApolloProvider (created by MyApp)
    in ApolloProvider (created by MyApp)
    in ApolloProvider (created by MyApp)
    in Container (created by MyApp)
    in MyApp (created by withApollo(App))
    in withApollo(App)
    in Suspense
```

**Comment by Edswor** [comment]
Add the pull request #229 for this.

**Comment by alexmiddeleer** [comment]
Thank you 😄

**Comment by alexmiddeleer** [comment]
Fixed in #249
